### PR TITLE
Address some problems detected by Codefactor

### PR DESCRIPTION
**Pull request by lidongze0629**
## Related issue number

Fixes

### CODE REVIEW

**Review by sighingnow — APPROVED** [review]

File: python/graphscope/dataset/ldbc.py [hunk 1]
```diff
@@ -23,254 +23,6 @@
...
-            ),
-            "studyAt",
-            ["classYear"],
-            src_label="person",
-            dst_label="organisation",
-        )
-    )
-    return graph
-"""
-
```
Comment (siyuan0322) [inline python/graphscope/dataset/ldbc.py:273]: Why delete these

File: python/jupyter/graphscope/graphscope_jupyter/_version.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
 # limitations under the License.
 #
 
+from graphscope.version import __version__
 from graphscope.version import __version_tuple__
 
 version_info = __version_tuple__
-__version__ = ".".join(map(str, version_info))
+__version__ = __version__
```
Comment (siyuan0322) [inline python/jupyter/graphscope/graphscope_jupyter/_version.py:23]: This line seems unnecessary?

File: python/graphscope/dataset/ldbc.py (hunk 1 above)
Comment (lidongze0629) [reply python/graphscope/dataset/ldbc.py:273]: These are comments

File: python/jupyter/graphscope/graphscope_jupyter/_version.py (hunk 2 above)
Comment (lidongze0629) [reply python/jupyter/graphscope/graphscope_jupyter/_version.py]: Remove it, thanks.

**Review by yecol — APPROVED** [review]

**Pull request merged**